src/collectd.conf.pod

```diff
@@ -1409,6 +1409,45 @@ Connection timeout in seconds. Defaults to B<2>.
 
 =back
 
+=head2 Plugin Connectivity
+
+connectivity - Documentation of collectd's C<connectivity plugin>
+
+
+  LoadPlugin connectivity
+  # ...
+  <Plugin connectivity>
+    Interface eth0
+  </Plugin>
+
+The C<connectivity plugin> queries interface status using netlink (man 7 netlink) which provides information about network interfaces via the NETLINK_ROUTE family (man 7 rtnetlink). The plugin translates the value it receives to collectd's internal format and, depending on the write plugins you have loaded, it may be written to disk or submitted to another instance.
+The plugin listens to interfaces configured in LoadPlugin (see configuration below).  If no interfaces are listed, then the default is for all interfaces to be monitored.
```

Comment:
>The plugin listens to interfaces configured in LoadPlugin

That is slightly wrong and needs to be rephrased/corrected.
Interfaces are configured in `<Plugin>` block, by use of `Interface` option, not 'in LoadPlugin', right?

Reply:
Clarified.